meritocracy/frontend/Makefile

```diff
@@ -14,6 +14,9 @@ unit:
 e2e:
 	@docker run --rm -v ${PWD}:/tmp -w /tmp node npm run e2e
 
+rest:
+	@../backend/rest_test.sh eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiZWRpdG9yIn0.tYUlheVyisdr0ezFYf92mC_dvSS02cpDvPBu9aKLySk
```

Comment:
what is this parameter you are passing, and what is the purpose of it?

Reply:
Its the token...